content/services/csv/powerdev_former_open_source_projects.csv

```diff
@@ -0,0 +1,15 @@
+"Alberta Speculation","Investigate multi-threaded speculation of alternative paths of execution in a sequential execution"
+"Diffusion magnetic resonance imaging of human brain","Drive diffusion basis spectrum imaging (DBSI) for clinical applications and large scale trials for a diffusion MRI modality which can increase the magnetic resonance imaging specificity for neurodegeneration diseases"
+"Drupal Testbots","Powered the testing and review of code contribution for Drupal 8 (qa.drupal.org)"
+"GCC","Contributed to improving support of various open source projects in the GCC family on POWER machines; began by tackling IBM Linux Technology Center items on bountysource.com"
+"`gdb`_","Worked on adding fast tracepoint support for ppc/ppc64/ppc64le"
```

Comment:
I wouldn't link "gdb" to the bug report. I would instead link "fast tracepoint support" to that URL.